presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetFileWriterFactory.java

```diff
@@ -97,6 +93,11 @@ public ParquetFileWriterFactory(
             return Optional.empty();
         }
 
+        ParquetWriterOptions parquetWriterOptions = ParquetWriterOptions.builder()
+                .setMaxPageSize(HiveSessionProperties.getParquetWriterPageSize(session))
```

Comment:
static import HiveSessionProperties.getParquetWriterPageSize and HiveSessionProperties.getParquetWriterBlockSize